packages/shorebird_cli/integration_test/shorebird_cli_integration_test.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
```

Comment:
why are we importing from flutter_tools? Is that needed over dart:io?

Reply:
Not sure why this is what vs code chose to import. Fixed.